### PR TITLE
A kaldi recipe based on the corpus named "aidatatang_200zh".

### PR DESCRIPTION
**Pull request by DatatangAI**
We have exposed a speech corpus named "aidatatang_200zh" on openslr recently, and this recipe is conducted to demonstrate that this corpus is a reasonable data resource for Chinese Mandarin speech recognition research. We hope the corpus could help much more researchers in Chinese speech recognition.

### CODE REVIEW

File: egs/aidatatang_200zh/s5/local/nnet3/tuning/run_tdnn_1a.sh [hunk 1]
```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+# This script is based on swbd/s5c/local/nnet3/run_tdnn.sh
+
+# this is the standard "tdnn" system, built in nnet3; it's what we use to
+# call multi-splice.
```
Comment (jtrmal) [inline egs/aidatatang_200zh/s5/local/nnet3/tuning/run_tdnn_1a.sh]: please have a look at some other chain/nnet3 scripts and add the performance indicators (dir_info and wers)

File: egs/aidatatang_200zh/s5/local/nnet3/run_tdnn.sh [hunk 2]
```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
```
Comment (jtrmal) [inline egs/aidatatang_200zh/s5/local/nnet3/run_tdnn.sh]: this should be only a symlink to a best-performing system in tuning.

File: egs/aidatatang_200zh/s5/local/nnet3/tuning/run_tdnn_2a.sh [hunk 3]
```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
```
Comment (jtrmal) [inline egs/aidatatang_200zh/s5/local/nnet3/tuning/run_tdnn_2a.sh]: please have a look at some other chain/nnet3 scripts and add the performance indicators (dir_info and wers)
Also probably your system should contain only 1 script in tuning/ unless you have two very different architenctures, say tdnn only, tdnn_lstm and tdnn_cnn

File: egs/aidatatang_200zh/s5/local/download_and_untar.sh [hunk 4]
```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
```
Comment (jtrmal) [inline egs/aidatatang_200zh/s5/local/download_and_untar.sh]: authors + license (Apache 2.0)

File: egs/aidatatang_200zh/s5/local/nnet3/tuning/run_tdnn_1a.sh (hunk 1 above)
Comment (DatatangAI) [reply egs/aidatatang_200zh/s5/local/nnet3/tuning/run_tdnn_1a.sh]: Thank you, I have generated and added corresponding performance indicators, please check them.

File: egs/aidatatang_200zh/s5/local/nnet3/run_tdnn.sh (hunk 2 above)
Comment (DatatangAI) [reply egs/aidatatang_200zh/s5/local/nnet3/run_tdnn.sh]: Yeah, I have turned it to be a symlink, please check it.

File: egs/aidatatang_200zh/s5/local/nnet3/tuning/run_tdnn_2a.sh (hunk 3 above)
Comment (DatatangAI) [reply egs/aidatatang_200zh/s5/local/nnet3/tuning/run_tdnn_2a.sh]: Thank you, I have added performance indicators in it, and removed /run_tdnn_1a.sh, please check them.

File: egs/aidatatang_200zh/s5/local/download_and_untar.sh (hunk 4 above)
Comment (DatatangAI) [reply egs/aidatatang_200zh/s5/local/download_and_untar.sh]: Thank you. I have completed the copyright part, please check it.

File: egs/aidatatang_200zh/s5/cmd.sh [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+# with slurm.  Different queues are configured differently, with different
+# queue names and different ways of specifying things like memory;
+# to account for these differences you can create and edit the file
+# conf/queue.conf to match your queue's configuration.  Search for
+# conf/queue.conf in http://kaldi-asr.org/doc/queue.html for more information,
+# or search for the string 'default_config' in utils/queue.pl or utils/slurm.pl.
+
+export train_cmd="queue.pl --mem 2G"
+export decode_cmd="queue.pl --mem 4G"
+export mkgraph_cmd="queue.pl --mem 8G"
```
Comment (jtrmal) [inline egs/aidatatang_200zh/s5/cmd.sh]: mkgraph_cmd can be probably removed (make sure by grepping for it)

File: egs/aidatatang_200zh/s5/local/chain/tuning/run_tdnn_2a.sh [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+
+# results
+# local/chain/compare_wer.sh exp/chain/tdnn_2a_sp/
+# Model                tdnn_2a_sp
+# WER(%)                     5.61
+# Final train prob        -0.0502
+# Final valid prob        -0.0913
+# Final train prob (xent)   -0.8047
+# Final valid prob (xent)   -1.0292
+
```
Comment (jtrmal) [inline egs/aidatatang_200zh/s5/local/chain/tuning/run_tdnn_2a.sh]: can you add tdnn_1a into the comparison?

File: egs/aidatatang_200zh/s5/local/format_data.sh [hunk 7]
```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+
+if [ -f ./path.sh ]; then . ./path.sh; fi
```
Comment (jtrmal) [inline egs/aidatatang_200zh/s5/local/format_data.sh]: could be shortened as 
```
[ -f ./path.sh ] && . ./path.sh
```
but I think path.sh should always be present

File: egs/aidatatang_200zh/s5/cmd.sh (hunk 5 above)
Comment (DatatangAI) [reply egs/aidatatang_200zh/s5/cmd.sh]: OK, I have removed it.

File: egs/aidatatang_200zh/s5/local/chain/tuning/run_tdnn_2a.sh (hunk 6 above)
Comment (DatatangAI) [reply egs/aidatatang_200zh/s5/local/chain/tuning/run_tdnn_2a.sh]: I have added several comparisons between different models. Please see list as below.

# results
# local/chain/compare_wer.sh exp/chain/tdnn_2a_sp
# Model                tdnn_2a_sp
# WER(%)                     5.61
# Final train prob        -0.0502
# Final valid prob        -0.0913
# Final train prob (xent)   -0.8047
# Final valid prob (xent)   -1.0292

# local/chain/compare_wer.sh --online exp/chain/tdnn_2a_sp
# Model                tdnn_2a_sp
# WER(%)                     5.61
# WER(%)[online]             5.69
# WER(%)[per-utt]            5.98
# Final train prob        -0.0502
# Final valid prob        -0.0913
# Final train prob (xent)   -0.8047
# Final valid prob (xent)   -1.0292

# local/chain/compare_wer.sh exp/chain/tdnn_1a_sp exp/chain/tdnn_2a_sp
# Model                tdnn_1a_sp tdnn_2a_sp
# WER(%)                     5.59      5.61
# Final train prob        -0.0488   -0.0502
# Final valid prob        -0.0925   -0.0913
# Final train prob (xent)   -0.8001   -0.8047
# Final valid prob (xent)   -1.0398   -1.0292

File: egs/aidatatang_200zh/s5/local/format_data.sh (hunk 7 above)
Comment (DatatangAI) [reply egs/aidatatang_200zh/s5/local/format_data.sh]: Yes, I have shortened it to ". ./path.sh".

**Comment by DatatangAI** [comment]
@jtrmal hi, how's it going?

**Pull request merged**